[CuriousCatBridge] Add new bridge

VerifiedJoseph opened the pull request:
Adds a bridge for [Curious Cat](https://curiouscat.me) profiles. Uses undocumented API that powers the website.

VerifiedJoseph committed:
Create CuriousCatBridge.php

VerifiedJoseph committed:
Add function formatUrls()

Converts URLs into html links with regex

VerifiedJoseph committed:
Update function formatUrls()

VerifiedJoseph committed:
Update description

VerifiedJoseph committed:
Update processContent()

teromene commented on bridges/CuriousCatBridge.php:
```diff
@@ -0,0 +1,121 @@
+
+	const CACHE_TIMEOUT = 3600;
+
+	private $urlRegex = '/https?:\/\/(?:www\.)?(?:[a-zA-Z0-9-.]{2,256}\.[a-z]{2,20})(\:[0-9]{2,4})?(?:\/[a-zA-Z0-9@:%_\+.~#?&\/\/=\-*]+|\/)?/';
```
You should cut this line to please Travis

VerifiedJoseph committed:
Use preg_replace() to convert URLs to links

Use preg_replace() from YoutubeBridge to convert URLs to links.
 https://github.com/RSS-Bridge/rss-bridge/blob/f76a23f/bridges/YoutubeBridge.php#L117-L119

VerifiedJoseph committed:
Remove extra '=' from formatUrls()

teromene commented:
Thanks for the PR !
